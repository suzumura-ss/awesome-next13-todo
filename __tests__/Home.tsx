import React from 'react';
import { render } from '@testing-library/react';
import Home from '@src/app/page';

describe(Home, () => {
  it('renders correctly', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
