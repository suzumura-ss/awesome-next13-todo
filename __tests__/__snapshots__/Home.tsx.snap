// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<div>
  <main>
    Awesome App
  </main>
</div>
`;
